### ISSUE TITLE
When specifying only Latin and Latin-extended why do we also get support for other languages in the file?

### ISSUE THREAD

**Issue by marko-stimac** [google/fonts#1189]
If I want to include Open Sans (400, 600) and Roboto (400,700) for Latin Extended in my project, even if I specify only Latin and Latin-extended I also get Vietnamese, Greek, Cyrillic and other languages in this generated url https://fonts.googleapis.com/css?family=Open+Sans:400,600|Roboto:400,700&amp;subset=latin-ext
Is that on purpose or a bug?

**Comment by davelab6** [comment]
@marko-stimac thanks for reporting this. It is on purpose, because you also get unicode-range rules so that you only download what the page needs (even if you didn't anticipate that it would.)

This is a duplicate of #68 #110 #327 #384 and perhaps others. 

We should add something to the GF FAQ to address this in order to close this issue :)

**Comment by alexgenaud** [comment]
Does the same "only as needed" logic also apply to other styles in the font-family? F.ex. if the API call includes "extra-bold" (?family=Font:400,700,800) but the page contains no bold text, will the bowser download Font-Bold?

**Comment by davelab6** [comment]
No, all major browsers now only downloade font faces when needed by the current page DOM :)
